### ISSUE TITLE
Product Vision & Product Mission

### ISSUE THREAD

**Issue by UmangBR** [amosproj/amos2022ss03-turtlebot-fleet-management#43]
## User story
1. As a team
2. We need a product vision and mission
3. So that we have clarity regarding the same

## Progress updates
* What i have done so far
* Step 1
* Step 2
* ...

## Acceptance criteria
* Everyone agrees on the product vision and mission 
* Inclusion in planning documents

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
